scripts/globals/weaponskills.lua

```diff
@@ -675,7 +675,7 @@ end
 xi.weaponskills.doPhysicalWeaponskill = function(attacker, target, wsID, wsParams, tp, action, primaryMsg, taChar)
     -- Set up conditions and wsParams used for calculating weaponskill damage
     local gearFTP = xi.combat.physical.calculateFTPBonus(attacker)
-    local gearAcc = math.floor(gearFTP * 10)
+    local gearAcc = math.ceil(gearFTP * 100)
```

Comment:
Add this TODO comment in all of this lines you changed:
`TODO: Separate gear fTP and acc bonuses`